ansible/roles/test/files/ptftests/inner_hash_test.py

```diff
@@ -0,0 +1,505 @@
+'''
+Description:    This file contains the inner hash test for SONiC
```

Comment:
@anish-n suggest to add supported topologies to the test description and also how to run example

Reply:
Will add, thanks!

Reply:
Added example of how to run in ecmp/test_inner_hashing.py, the supported topologies can be found in pytestmark as per the standard